packages/react-formio/src/components/tabs/tabs.component.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import { fireEvent, render } from "@testing-library/react";
+// import userEvent from "@testing-library/user-event";
+import { Sandbox } from "./tabs.component.stories";
+import { TabsItemProps } from "./tabs.component";
+
+describe("tabs", () => {
+
+  it("should display the tabs component", () => {
+    const { getByTestId, queryAllByTestId } = render(<Sandbox {...Sandbox.args} />);
+
+    const tabsComponent = getByTestId("tabs-comp");
+    const allButtonsTab = queryAllByTestId("button-tab");
+
+    expect(tabsComponent).toBeInTheDocument();
+    allButtonsTab.forEach((buttonTab) => {
```

Comment:
alors cet test ne valide rien.

Imaginons qu'il n'y ai pas d'items passé au tabs, et bien ton test sera tjrs OK... Il faut absolument que ton test vérifie la présence d'un contenu de type text. Evite de faire des itérations dans ton test unitaire. Il faut etre déclaratif!

Reply:
exact je vais rectifier merci :)